src/N3Lexer.js

```diff
@@ -429,6 +429,11 @@ export default class N3Lexer {
     return err;
   }
 
+  _ignoreStartingBom(input) {
+    const bom = /^(\ufeff?)/.exec(input);
+    return input.substr(bom[1].length, input.length);
```

Comment:
```suggestion
   if (input.startsWith('\ufeff')) …
```

to avoid unnecessarily subsetting (very large) strings.